inc/saiacl.h

```diff
@@ -816,7 +816,7 @@ typedef enum _sai_acl_table_attr_t
     /**
      * @brief Range type defined
      *
-     * @type sai_acl_range_type_t
+     * @type sai_s32_list_t sai_acl_range_type_t
```

Comment:
so if you set for example L4 src port range and outer vlan, and then you add acl entry which is range packet length what then ?